### PR TITLE
Minor Dice Loss adjustments

### PR DESCRIPTION
**Pull request by zoq**
- include Dice loss into the build routine
- minor style fixes

### CODE REVIEW

**Comment by zoq** [comment]
Since the style check is working again, there are some issues we missed, will fix those in another PR.

File: src/mlpack/methods/ann/loss_functions/dice_loss.hpp [hunk 1]
```diff
@@ -56,7 +51,7 @@ class DiceLoss
   /**
    * Create the DiceLoss object.
    *
-   * @param smooth The parameter to avoid overfitting.
+   * @param smooth The laplace smoothing parameter.
```
Comment (rcurtin) [inline src/mlpack/methods/ann/loss_functions/dice_loss.hpp]: Should this be `Laplace`?  I know that is really pedantic though. :)

**Comment by zoq** [comment]
Looks like https://github.com/mlpack/mlpack/pull/1713 was faster, great.

File: src/mlpack/methods/ann/loss_functions/dice_loss.hpp (hunk 1 above)
Comment (zoq) [reply src/mlpack/methods/ann/loss_functions/dice_loss.hpp]: Nice catch!

**Pull request merged**